clang/test/3C/unwritable_rootcauses.h

```diff
@@ -0,0 +1,15 @@
+
+   
```

Comment:
I guess my reasoning is this is the core functionality that helped me write the code, so if a regression happens it may be useful to have the isolated core as a test.

Reply:
That may have been your development process, but at this point, I don't see what makes these cases any more "core" or otherwise different in nature as a group than the cases in `root_cause.c`.  ISTM, if there's a regression affecting some of these cases and/or some of the cases currently in `root_cause.c`, then each broken case will cause a diagnostic verification error; the only difference is whether all of the errors show up in `root_cause.c` or some show up in each file.  Either way, we can pick any one of the errors to start the investigation of what broke, so I don't see a practical difference in difficulty of troubleshooting.  And as I mentioned, the advantage of using `root_cause.c` is that we already have the infrastructure there to verify that these are ordinary root causes of wildness when the file is writable; in fact, if these cases are not yet covered by `root_cause.c`, we probably want to have tests of their ordinary root-cause messages, regardless of any test of unwritability.

Reply:
Hm.  I was assuming you would move these test cases to `root_cause.c` (not delete them) if you thought there was value in testing the affected pointer counts or the test cases covered situations that weren't already adequately covered by `root_cause.c`.  It looks like `root_cause.c` already has a `void *x` and a `void void_star_fn(void *p)`, but the `void *f1(void)` and the array might be worth adding.  If you want, you can change the code that manipulated the affected pointer counts to use existing elements in `root_cause.c`.

Reply:
Resolving per https://github.com/correctcomputation/checkedc-clang/pull/665#issuecomment-897684686